sp/src/game/server/ez2/npc_zassassin.cpp

```diff
@@ -731,37 +739,6 @@ void CNPC_Gonome::IdleSound( void )
 	EmitSound( filter, entindex(), "Gonome.Idle" );	
 }
 
-//=========================================================
-// PainSound 
-//=========================================================
-void CNPC_Gonome::PainSound( const CTakeDamageInfo &info )
-{
-	// Burning creatures shouldn't play pain sounds constantly
-	if ( !( IsOnFire() && info.GetDamageType() & DMG_BURN ) )
-	{
-		CPASAttenuationFilter filter( this );
```

Comment:
CNPC_Gonome uses this CPASAttenuationFilter while CNPC_BasePredator does not. What is the difference exactly?

Reply:
`EmitSound`'s usage on `CNPC_BasePredator` uses the simpler, default override of `EmitSound`. On the inside, it has its own `CPASAttenuationFilter` which is written identically.